### PR TITLE
Added readme

### PR DESCRIPTION
**Pull request by anacierdem**
Some help :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# n64chain
+
+This is a Nintendo 64 development toolchain based on GCC that do not depend on any proprietary Nintendo library.
```
Comment (tj90241) [inline README.md]: that does

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The library includes basic OS functionality supporting multi-threading through hardware interrupts on the
+single MIPS 4300 CPU of the N64. Mainly written in C and MIPS assembly, it is expected to provide a
+lightweight development environment and thus requires some low-level baby sitting.
+
+## Building
+
+To build the toolchain, first put your boot code named `header.bin` on the `libn64` folder. Run `build-linux64-toolchain.sh`
+in the `tools` folder on a bash-compatible shell to start building the cross-compiler.
+Prerequisites are MPFR and MPC with development headers, which can be installed via `apt install libmpfr-dev libmpc-dev`
```
Comment (tj90241) [inline README.md]: `build-essential` (Debian/Ubuntu) or whatever the Redhat equivalent is.
Also, `libgmp-dev`, `yacc`, and `bison`.

File: README.md (hunk 1 above)
Comment (anacierdem) [reply README.md]: Nice catch!

**Comment by anacierdem** [comment]
Updated.

**Comment by tj90241** [comment]
Thanks!

**Pull request merged**